packages/aws-lambda/test/integration_test/test.js

```diff
@@ -1010,74 +1024,84 @@ function registerTests(handlerDefinitionPath) {
   }
 
   function verifyLambdaEntry(spans, expectations) {
-    return expectExactlyOneMatching(spans, span => {
-      if (expectations.parent) {
-        expect(span.t).to.equal(expectations.parent.t);
-        expect(span.p).to.equal(expectations.parent.s);
-      } else {
-        expect(span.t).to.exist;
-        expect(span.p).to.not.exist;
-      }
-      expect(span.s).to.exist;
-      expect(span.n).to.equal('aws.lambda.entry');
-      expect(span.k).to.equal(constants.ENTRY);
-      expect(span.f).to.be.an('object');
-      expect(span.f.h).to.not.exist;
-      expect(span.f.hl).to.be.true;
-      expect(span.f.cp).to.equal('aws');
-      expect(span.f.e).to.equal(qualifiedArn);
-      expect(span.async).to.not.exist;
-      expect(span.error).to.not.exist;
-      expect(span.data.lambda).to.be.an('object');
-      expect(span.data.lambda.arn).to.equal(qualifiedArn);
-      if (expectations.alias) {
-        expect(span.data.lambda.alias).to.equal('anAlias');
-      } else {
-        expect(span.data.lambda.alias).to.not.exist;
-      }
-      expect(span.data.lambda.runtime).to.equal('nodejs');
-      if (expectations.trigger) {
-        expect(span.data.lambda.trigger).to.equal(expectations.trigger);
-      }
-      const { error } = expectations;
-      if (error && error.startsWith('lambda')) {
-        expect(span.data.lambda.error).to.be.a('string');
-        expect(span.data.lambda.error).to.include('Boom!');
-        expect(span.ec).to.equal(1);
-      } else if (error === 'http') {
-        expect(span.data.lambda.error).to.equal('HTTP status 502');
-        expect(span.ec).to.equal(1);
-      } else if (error === false) {
-        expect(span.data.lambda.error).to.not.exist;
-        expect(span.ec).to.equal(0);
-      } else {
-        throw new Error(`Unknown error expectation type, don't know how to verify: ${error}`);
-      }
-      verifyHeaders(span);
-    });
+    const checks = [
```

Comment:
Most of this diff comes from migrating from the `expectExactlyOneMatching(spans, callback)` syntax to `expectExactlyOneMatching(spans, [expectations])` syntax.